README.md

```diff
@@ -100,7 +100,8 @@ jovian pull    # Pull the latest changes
 
 For feedback, suggestions and feature requests, drop us a line at hello@jvn.io or create a ticket in the [issues tab](https://github.com/jvn-io/jovian-py/issues) .
 
-## Development and Testing
+## Contributors
 
-To run the tests, run the following command in the project root:  
-`python -m unittest discover` [`-v` for verbose]
+Aakash N S
```

Comment:
You should probably use bullet list for this. Otherwise they will all show up in a row.

```
- Aakash N S
- Prajwal
- Sid
```